src/lib/Toast/Toast.svelte

```diff
@@ -172,20 +189,24 @@
   .success {
     color: var(--toast-success-text, #fff);
     background-color: var(--toast-background-color, #24aa5a);
+    border-left: var(--toast-success-border-left);
   }
 
   .info {
     color: var(--toast-info-text, #fff);
     background-color: var(--toast-background-color, #87ceeb);
+    border-left: var(--toast-info-border-left);
   }
 
   .warn {
     color: var(--toast-warn-text, #fff);
     background-color: var(--toast-background-color, #f3a42d);
+    border-left: var(--toast-warn-border-left);
   }
 
   .error {
     color: var(--toast-error-text, #fff);
     background-color: var(--toast-background-color, #f04438);
+    border-left: var(--toast-error-border-left);
```

Comment:
Can we add default value for warn/info/error/success, so that we don't require to define css variables.

Reply:
currently border left required only in lighthouse and this component used in other repo and have different usecase